TesSense.py

```diff
@@ -13,12 +13,16 @@
  Add: ability to find TP-Link devices on the network and control them.
 """
 
+import asyncio
+import sys
+import logging
 import datetime
 from time import sleep
 
-rate = minrate = 0                                    # Minimum rate you want to set the charger to
-volts = 120                                           # Minimum volts, until detected by the charger
-pshown = loop = charge = charging = False             # init more variables
+# Set stdout as logging handler
```

Comment:
Not required, but since SenseLink uses logging, you'll need this to have it echo out on stdout